api.py

```diff
@@ -256,6 +242,9 @@ def update_cache(cls, data):
         now = datetime.now()
         cls._last_request = now
 
+    def data_received(self, chunk):
```

Comment:
What's the purpose of this method?

Reply:
It's there to silence a warning about unimplemented abstract methods.

Reply:
Oh, there's warnings nowadays? Fair enough then.

I haven't touched the code in a while, that's why I'm asking.

Reply:
It was just PyCharm complaining, nothing related to the python runtime in particular.